### ISSUE TITLE
Project update

### ISSUE THREAD

**Issue by wparkermackenzie** [LoRaWanMiniMouse/Mini-Mouse#10]
Can you provide an update on the project? If you could provide what is being actively developed and what you expect for a near term roadmap would be great. Looking at this project it seems like a great alternative to the existing stack. Knowing your development plans will help me to determine which stack to port and to minimize duplication of development efforts. Thank you in advance, Parker

**Comment by vanthome** [comment]
I also agree, it would be nice to get a roadmap of where this project is heading. I'm really confused by all the branches meanwhile, I don't see where to follow the development!

**Comment by vanthome** [comment]
Hi, project maintainers, can you please answer this question? We are using this code and would like to know the status.

**Comment by vanthome** [comment]
@fholin can you please make a statement here? We are users of the stack and want to know where it will go?

**Comment by LoRaWanMiniMouse** [comment]
At the end of this year we should deliver an update supporting some services. 
BR

**Comment by vanthome** [comment]
That's nice to hear it works however already for our device. We just need a stable master branch (currently there are countless of branches and it's unclear what they are good for). Also an important question to us is support for LoRaWAN 1.1 is this even planned with this mac?

**Comment by vanthome** [comment]
@LoRaWanMiniMouse can you pls. give an update about the project status?

**Comment by vanthome** [comment]
@LoRaWanMiniMouse can you pls. give an update about the project status?
